### DISCUSSION TITLE
ArcGIS tile service

### DISCUSSION THREAD

**Discussion by sro-kciope** [maplibre/maplibre-native#2057] [Q&A]
Hello everyone, 

I developped and android app which have some map feature related, using maplibre for this part I have to change my tile provider from Jawg.io which worked perfectly out of box with maplibre and now trying to connect to ArcGIS basemap-api I can't get any tile to show on my app. I get in the json value of my mapView that : 
```
Failed to load the map with : {"version":8,"sprite":"https://cdn.arcgis.com/sharing/rest/content/items/7dc6cea0b1764a1f9af2e679f642f0f5/resources/styles/../sprites/sprite?token=**mysuperapikey**","glyphs":"https://basemaps-api.arcgis.com/arcgis/rest/services/World_Basemap_v2/VectorTileServer/resources/fonts/{fontstack}/{range}.pbf?token=**mysuperapikey** [...]
``` 

And get log from : 
```
Mbgl-HttpRequest my.app.name  E  [HTTP] Unable to parse resourceUrl tile/4/5/8.pbf
```  

Has anyone else ever tried to connect this tile provider ?

Any answer would be good :)

Maplibre android sdk version 10.2.0

**Comment by HarelM** [comment]
Are you using maplibre web, maplibre native or mapbox?

**Reply by sro-kciope** [reply]
Good question, i wrote mapbox by reflex.
Using maplibre android sdk* @HarelM

**Reply by HarelM** [reply]
I'll move this to the native repo then...

**Reply by barbaramartina** [reply]
@HarelM  do you have a link for the moved topic? I am trying to integrate a different tiles provider as well and facing some issues. Thanks

**Reply by HarelM** [reply]
This is the moved issue, it's under the native repo now.
